communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -137,3 +287,9 @@ tabs:
       https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html
     button_tip: View tutorial
     button_icon: tutorial
+
+- id: learning-pathway
+  title: Learning pathways
+  heading_md: >
+      {gtn modal}[Microbiome learning pathways](https://training.galaxyproject.org/training-material/learning-pathways/metagenomics.html) on the GTN.
```

Comment:
What is this `{gtn modal}`?

Reply:
Idont know but it is used sometimes when adding a GTN link in the lab, I will try to remove it and see what will happen

Reply:
Now I know its meaning, this allow it to open as a pop up within Galaxy not in a new tab, I will return it back again